### DISCUSSION TITLE
Communications channels dedicated for Transmission project contributors

### DISCUSSION THREAD

**Discussion by nevack** [transmission/transmission#3391] [General]
Are there any means of communications dedicated to the Transmission project?
Like Slack/Telegram/IRC?

Maybe it's possible to enable Team discussions for Transmission team on GitHub?

Thanks!

P.S. Initially I have tried to contact @ckerr via `<first name> at <first name><last name>.com`, but haven't received any reply :(

**Comment by ckerr** [comment]
@nevack sorry for my lack of reply. It's not personal; I'm just being bad at mail. :disappointed:  Probably better to keep it on GitHub for now.

I've enabled Team discussions; https://github.com/orgs/transmission/teams/contributors/discussions should be working now.

If there's anyone who ought to be included in the contributor group who's not already there, please lmk

**Reply by GaryElshaw** [reply]
404 not found

**Reply by ckerr** [reply]
How about now

**Reply by GaryElshaw** [reply]
Still no-go. It could be old-school, 'we update every 24 hours for users on x-plan'.

**Reply by ckerr** [reply]
You have to accept the invitation to the contributor group first.

**Comment by Pentaphon** [comment]
I second keeping everything on Github. Anything else will just fracture an already small community that is just finally pulling together after moving to Github. Still 404 not found for me as well, though.
